src/components/ListItems/ListItems.component.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import styled from 'styled-components';
+import Paper from '@material-ui/core/Paper';
+import videos from '../../data/youtube-videos-mock.json';
+
+/**
+ *Styled components used in all the component
+ */
+
+const StyledMainContainer = styled.div`
+  margin-top: 1rem;
+`;
+
+const StyledListGrid = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+`;
+
+const StyledListSingleCard = styled(Paper)`
+  disfplay: flex;
+  flex-direction: column;
+  width: 21%;
+  margin-left: 2%;
+  margin-right: 2%;
+  margin-bottom: 2rem;
+  cursor: pointer;
+  @media only screen and (max-width: 900px) {
+    width: 45%;
+    margin-left: 2.5%;
+    margin-right: 2.5%;
+  }
+
+  @media only screen and (max-width: 600px) {
+    width: 90%;
+    margin-left: 5%;
+    margin-right: 5%;
+  }
+`;
+
+const StyledListVideoImage = styled.img`
+  width: 100%;
+`;
+
+const StyledInfoContainer = styled.div`
+  width: 100%;
+  height: 13rem;
+  overflow-hidden;
+`;
+
+const StyledListVideoTitle = styled.h2`
+  width: 100%;
+  font-size: 1.1rem;
+  padding-left: 5%;
+  padding-right: 5%;
+  overflow-wrap: anywhere;
+  text-align: left;
+  margin-top: 0;
+  margin-bottom: 0.5rem;
+  cursor: pointer;
+`;
+
+const StyledListVideoDescription = styled.p`
+  width: 100%;
+  font-size: 0.8rem;
+  padding-left: 5%;
+  padding-right: 5%;
+  overflow-wrap: anywhere;
+  text-align: left;
+  color: gray;
+  margin: 0;
+`;
+
+/**
+ *@desc Single video element card with information of the videos
+ */
+function SingleElement({ title, description, thumbnails, role }) {
```

Comment:
Now that you are using props, it could be useful to put in place a set of rules for those props, so that the component expects exactly what you meant. Check the documentation for these and let me know if you have any questions:

https://reactjs.org/docs/typechecking-with-proptypes.html